xapian-letor/Makefile.am

```diff
@@ -108,4 +134,4 @@ noinst_HEADERS +=\
 	common/str.h\
 	common/stringutils.h
 
-DISTCHECK_CONFIGURE_FLAGS = "XAPIAN_CONFIG=$(XAPIAN_CONFIG)"
+DISTCHECK_CONFIGURE_FLAGS = "XAPIANLETOR_CONFIG=$(XAPIANLETOR_CONFIG)"
```

Comment:
This change seems wrong - `xapian-letor` doesn't use `xapianletor-config`, but it ought to use the in-tree xapian-config for the nested build when running `make distcheck`.